### PR TITLE
[WRF] Update wind derivation to split out cos/sin components

### PR DESCRIPTION
**Pull request by saskatchuwan**
Update our wind direction computation to split out sin and cos components - code extracted from source: https://github.com/UrbanSystemsLab/climateiq-cnn/pull/114

### CODE REVIEW

File: usl_pipeline/cloud_functions/main_test.py [hunk 1]
```diff
@@ -636,9 +636,9 @@ def test_compute_custom_wps_variables_wind():
     processed_ds = main._compute_custom_wps_variables(ds)
 
     # Check that processed dataset contains newly computed variables
-    assert all(var in processed_ds.keys() for var in ["WSPD10", "WDIR10"])
+    assert all(var in processed_ds.keys() for var in ["WSPD", "WDIR_SIN", "WDIR_COS"])
```
Comment (waltaskew) [inline usl_pipeline/cloud_functions/main_test.py:639]: should we have some tests for the computed values as well?

**Review by waltaskew — APPROVED** [review]
hooray for trigonometry!

File: usl_pipeline/cloud_functions/main_test.py (hunk 1 above)
Comment (saskatchuwan) [reply usl_pipeline/cloud_functions/main_test.py:639]: @MadhaviJain90 is this something you can help validate?

Comment (waltaskew) [reply usl_pipeline/cloud_functions/main_test.py:639]: can you use some simpler wind vectors as test inputs? like, one that runs exactly east?

Comment (MadhaviJain90) [reply usl_pipeline/cloud_functions/main_test.py:639]: The only thing that is concerning to me is that all values of sin and cos are always negative? The scaling was from 1 to -1. So we should be seeing some positive values too

Comment (waltaskew) [reply usl_pipeline/cloud_functions/main_test.py:639]: although not knowing exactly what they represent, I'd assume you'll get simpler values to reason about with unit vectors like [1, 0, 0] & such

Comment (saskatchuwan) [reply usl_pipeline/cloud_functions/main_test.py]: Thanks for the suggestion. updated the test. feels right but can't confirm my trig is bad
@MadhaviJain90 take a look at the expected values in test assertions. if it looks good, i'll merge!

**Review by MadhaviJain90 — APPROVED** [review]
All good 👍

File: usl_pipeline/cloud_functions/main_test.py (hunk 1 above)
Comment (saskatchuwan) [reply usl_pipeline/cloud_functions/main_test.py]: Perf!

**Pull request merged**